torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3224,15 +3289,20 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.min,
            variant_test_name='reduction_no_dim',
            dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
-           dtypesIfCPU=all_types_and(torch.float16, torch.bfloat16, torch.bool),
-           dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            supports_out=False,
            sample_inputs_func=sample_inputs_max_min_reduction_no_dim,),
+    OpInfo('mm',
```

Comment:
This OpInfo and its sample inputs look good. You may want to separate it into another PR because it could be merged now.

Reply:
Opened #55446. Thanks!